<launch>
  <node type="fiducial_slam.py" pkg="fiducial_slam" output="screen" name="fiducials_slam">
    <param name="map_file" value="$(env HOME)/.ros/slam/map.txt" />
    <param name="obs_file" value="$(env HOME)/.ros/slam/obs.txt" />
    <param name="trans_file" value="$(env HOME)/.ros/slam/trans.txt" />
    <param name="mapping_mode" value="true" />
    <param name="camera_frame" value="raspi_camera_frame" />
  </node>
</launch>
